conjure-java-core/src/main/java/com/palantir/conjure/java/services/UndertowServiceInterfaceGenerator.java

```diff
@@ -83,8 +83,7 @@ private MethodSpec generateServiceInterfaceMethod(
 
         ServiceGenerator.getJavaDoc(endpointDef).ifPresent(content -> methodBuilder.addJavadoc("$L", content));
 
-        endpointDef.getReturns().ifPresent(type -> methodBuilder.returns(
-                UndertowTypeFunctions.unbox(returnTypeMapper.getClassName(type))));
```

Comment:
Should be safe to delete this `unbox` method too now right?

Reply:
I thought I had, oops!

Reply:
Updated